src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1523,6 +1571,11 @@ export class AmpStoryPlayer {
       case STORY_MESSAGE_STATE_TYPE.MUTED_STATE:
         this.onMutedStateUpdate_(/** @type {string} */ (data.value));
         break;
+      case STORY_MESSAGE_STATE_TYPE.UI_STATE:
+        if (this.isDesktopPanelExperimentOn_) {
+          this.onUiStateUpdate_(/** @type {number} */ (data.value));
```

Comment:
Just to make sure I follow: this is used for a window resized while the story is displayed, and the `getUiState_()` is used for Story to Story navigations?

Reply:
Exactly. I added a comment for that.